recognizing_contributors_sponsors_and_partners (#261)
mautic/mautic-community-handbook: docs/policies/recognizing_contributors_sponsors_and_partners.rst

7. Each initiative/team/working group that is recognizing or promoting the work from current significant contributors in a release must state that it is doing so as part of each release planning meeting, and must provide the list of contributors to be recognised to the release leader at least one week prior to the GA release date.

8. Each initiative/team/working group's leadership chain (leads and assistant leads) are responsible for vetting, and potentially filtering, the current significant contributor list in order to conform to the overall goal and community norms of this policy.

9. Persons or companies who believe they are being unfairly precluded from being identified as a current significant contributor can contact the Project Lead to discuss and, where appropriate, resolve the issue.

> 10. Individuals or companies so recognized may promote that fact, publish that they have been recognized, and link to the recognition. Such an individual or company may not, however, imply any endorsement of their product(s) or service(s) by Mautic or others based on such recognition.

Reviewer: ```suggestion
10. Individuals or companies so recognized may promote that fact, publish that they have been recognized, and link to the recognition. Such an individual or company may not, however, imply any endorsement of their products or services by Mautic or others based on such recognition.
```

Recognizing contributors, sponsors and partners
###############################################

Background information 
======================

.. vale off

The Mautic Community wishes to encourage initiative/team/working group teams to acknowledge and thank their sponsors and contributors,
but without the \*.mautic.org web properties or the Mautic application 
becoming overly cluttered with advertising or promotional messages.

Additionally, the community ensures that companies don't ‘game’ the system 
by providing minimal contributions to be recognized (e.g. with logo placement/links).

.. vale off

Recognizing that the community norms around such messages are continually 
changing and that it's probably impossible to codify a process precisely, 
the recommendation is that we establish a policy framework for the Community 
Leadership Team to use when evaluating each situation, and leave the final 
decisions in the case of any disputes to the Community Council. 

Definitions
-----------

> vale off

Reviewer 2: ```suggestion
.. vale off
```

Please ensure you render the docs in the front-end in Gitpod (use this icon)

![recognizing_contributors_sponsors_and_partners-rst-—-mautic-community-handbook-—-Gitpod-Code-02-05-2025_02_03_PM](https://github.com/user-attachments/assets/6a526ac4-bb67-4523-832c-281cb7c4b30a)

Then you can pick up things like this easily!

![Preview-recognizing_contributors_sponsors_and_partners-rst-—-mautic-community-handbook-—-Gitpod-Code-02-05-2025_02_02_PM](https://github.com/user-attachments/assets/178f84a5-1b8f-45df-a704-579eeee234c5)

**Contributor**: there are many way to contribute to Mautic projects ranging from 
committing code to writing good bug reports to helping with documentation to 
doing community outreach to funding developers to work on the project. Even 
organizations and individuals that are "merely" adopting Mautic can be extremely useful 
contributors if they provide good, detailed, and timely feedback about requirements and bugs.

**Individual Contributor**: a contributor who is a person.

**Organization Contributor**: an organisation that employs a one or more people who are contributors.

> **Significant Contribution**: a significant contribution directly impacts the outcome of the project, not just its size.. Typically larger 

Reviewer 2: ```suggestion
**Significant Contribution**: a significant contribution directly impacts the outcome of the project, not just its size. Typically larger 
```

Small fix, duplicated full stop.

> contributions are more significant, but that doesn't preclude an extremely valuable

Reviewer 2: This seems like an incomplete sentence - the original text is:

> Typically larger contributions are more significant, but that does not preclude an extremely valuable two line fix to a critical, but difficult to find, bug from being "significant". Nor does it preclude a giant code dump of rarely used code from being "not significant". As a general rule of thumb, somewhere between 25% and 50% of the contributions to a project are significant.

As policies are extremely important only changed via debate and public consultation in the Council/General Assembly, please take great care to ensure that words are an exact match.

A two-line fix to a critical but difficult-to-find bug from being "significant." Nor does it preclude a giant code dump of rarely used code from being "not significant."
As a general rule of thumb, somewhere between 25% and 50% of the contributions to a project are significant.

**Current Contribution**: a current contribution refers to one included in the latest major release.
cycle, for example, if the project is working on release 4.0, then a current contribution is a contribution whose code or effect is still significant in the 4.0 code base. The currency of a contribution fades with time, even if that contribution is still part of the code base. For example, bug fixes to release 1.0 are probably no longer "current" when the current major release is 4.0.

**Current Significant Contributor**: a Contributor who has made one or more Significant Contributions that are also Current Contributions.

> vale off

Reviewer 2: ```suggestion

.. vale off

```

See note above, please check your work in the browser preview to ensure that the content reads properly! The vale on/off also requires a line break either side of it, which I've added.

3. Infrastructure sponsors who are providing a service used by the Mautic Community at free or discounted price will have a logo and link to the most relevant page on their website for as long as the membership is active, listed at :xref:`Mautic.org/sponsors`.
   
4. If a sponsor cancels their sponsorship or the service is no longer being used, the link will be removed as soon as notification of cancellation is received

5. Event sponsors will be listed on the event websites, and will not be eligible to be listed on mautic.org unless they are a regular monthly sponsor in addition to supporting the event.
   
6. Partners will be announced on social media and email when they become a partner, announcing their joining of the programme.
   
7. Partners will be featured in the email template of the Mautic Community Newsletter with a logo and link to their partners landing page.
   
8. The top three partners each month will be listed on the mautic.org homepage with a link to their partners landing page.
   

Credits
#######

This policy is based on those used by several other Open Source communities including:

> Eclipse: https://www.eclipse.org/org/documents/Recognizing_Project_Contributions_Policy_2008.pdf

Reviewer 2: These need to be link files please. you can use the name (e.g. Eclipse Foundation contributor recognition policy) as the link text. Same for the second link.

Recognizing contributors, sponsors and partners
###############################################

Background information 

> ======================

Reviewer 2: Please take care of your headings, this is a H3 and it should be a H2. This will need checking and addressing throughout the document.

```
H1: ###
H2: ***
H3: ===
H4: ---
H5: ~~~
```

2. Sponsors who are contributing under $100/mth will have their name or organization name listed on :xref:`the Mautic Sponsor page`.

3. Infrastructure sponsors who are providing a service used by the Mautic Community at free or discounted price will have a logo and link to the most relevant page on their website for as long as the membership is active, listed at :xref:`the Mautic Sponsor page`.
   
4. If a sponsor cancels their sponsorship or the service is no longer being used, the link will be removed as soon as notification of cancellation is received

5. Event sponsors will be listed on the event websites, and will not be eligible to be listed on mautic.org unless they are a regular monthly sponsor in addition to supporting the event.
   
6. Partners will be announced on social media and email when they become a partner, announcing their joining of the programme.
   
7. Partners will be featured in the email template of the Mautic Community Newsletter with a logo and link to their partners landing page.
   
8. The top three partners each month will be listed on the mautic.org homepage with a link to their partners landing page.
   

> Credits

Reviewer 2: ```suggestion
Credits and useful resources
```

All headings must be unique and the build is failing because Credits is used elsewhere as a heading. Please also make sure to update the underline.
